cheerlights/app.go

```diff
@@ -1,13 +1,30 @@
 package main
 
-import . "github.com/alexellis/blinkt_go/sysfs"
+import (
+	"os"
+	"strconv"
+
+	. "github.com/alexellis/blinkt_go/sysfs"
+)
+
+func getEnv(envVar string, assumed int) int {
```

Comment:
We should probably return time.Duration here.

Reply:
`Delay` is expecting an `Int`

Reply:
OK fair point. We should probably change the library to be more "Go"